lib/Driver/ToolChains.cpp

```diff
@@ -1037,7 +1038,11 @@ static void getRuntimeLibraryPath(SmallVectorImpl<char> &runtimeLibPath,
     runtimeLibPath.append(programPath.begin(), programPath.end());
     llvm::sys::path::remove_filename(runtimeLibPath); // remove /swift
     llvm::sys::path::remove_filename(runtimeLibPath); // remove /bin
-    llvm::sys::path::append(runtimeLibPath, "lib", "swift");
+    if (shared) {
+      llvm::sys::path::append(runtimeLibPath, "lib", "swift");
+    } else {
+      llvm::sys::path::append(runtimeLibPath, "lib", "swift_static");
+    }
```

Comment:
I think this is better as a ternary:

    llvm::sys::path::append(runtimeLibpath, "lib", shared ? "swift" : "swift_static");